src/components/RevievCard.jsx

```diff
@@ -19,23 +25,39 @@ useEffect(() => {
     })
 }, [id])
 
-    return (
-        <section className="borderElem">
-            {isLoading ? (
-          <p>Loading...</p>
-        ) : (
-            <> 
-        <h1>ReviewCard </h1>
-        {revCard && <h2>Title: {revCard.title}</h2>}
-        {revCard &&<p> : {revCard.review_body}</p>}
-
-        {revCard &&<h2>Owner : {revCard.owner}</h2>}
-        {revCard &&<h2>Id : {revCard.review_id}</h2>}
-
-        {revCard &&<h2>Category : {revCard.category}</h2>}
-        {revCard &&<img src={revCard.review_img_url} alt="Review Image" />}
-        </>
-        )}
+useEffect(() => {
+    fetchComments(id)
+        .then(({ review })  => {
+            console.log(review)
+           setComments(review)
+
+    })
+}, [id])
+
+return (
+    <section className="borderElem">
+        {isLoading ? (
+      <p>Loading...</p>
+    ) : (
+        <> 
```

Comment:
No need for the snippet here as the whole expression which starts on line 39 is a child of <section>